### PR TITLE
Create ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58128](https://cumulocity.atlassian.net/browse/MTM-58128)
Original PR: [5707](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5707)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/cd] (#5707)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix Issue Switching Tabs with Dashboard Date Context Widgets
```

File: content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58128
+version: 1019.7.3
+---
+Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/cd] (#5707)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:17]: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where users were unable to switch between dashboard tabs when the dashboard contained two widgets using dashboard date contexts. Users can now smoothly transition between tabs regardless of widgets using dashboard date contexts.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix inability to switch tabs with multiple widgets using dashboard date context
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:3]:
```suggestion
title: Switching tabs with multiple widgets using the dashboard date context works smoothly
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**